src/activate/tmp_files.rs

```diff
@@ -0,0 +1,24 @@
+use super::ActivationResult;
+use std::process;
+
+type TmpFilesActivationResult = ActivationResult<process::ExitStatus>;
+
+pub fn activate() -> TmpFilesActivationResult {
+    let mut cmd = process::Command::new("systemd-tmpfiles");
+    cmd.arg("--create");
+    let status = cmd
+        .stdout(process::Stdio::inherit())
+        .stderr(process::Stdio::inherit())
+        .status();
+    Ok(status.unwrap())
+}
+
+pub fn deactivate() -> TmpFilesActivationResult {
+    let mut cmd = process::Command::new("systemd-tmpfiles");
+    cmd.arg("--clean").arg("--remove");
```

Comment:
I think we need to pass the config file generated by system-manager here, so that we only clean up that one and leave the ones belonging to the host distro in place.